wagtail/utils/text.py

```diff
@@ -0,0 +1,9 @@
+try:
+    from unidecode import unidecode
+except ImportError:
+    def unidecode(value):
+        return value
+
+
+def string_to_ascii(value) -> str:
+    return str(unidecode(value))
```

Comment:
I don't think this is right... If unidecode isn't installed, this function will have no effect and the string will remain as unicode, breaking any code that relies on it returning ASCII.

Reply:
Yeah you are right, I think we can reuse some logic from the `cautious_slugify` code

Reply:
I reused the fallback mechanism which was in the postgres search module. I now removed that via a force-push since unidecode is required anyway.

The idea of this PR is purely to focus on consolidating all calls to unidecode, but still leverage it

Reply:
Ah, OK :-) In that case, I suggest we hold off merging this until we have a fix for #3088 in place, since this extra level of indirection makes it less obvious that wagtailforms cannot (currently) be migrated away from unidecode without losing existing data.